docs/core-concepts/model-basics.md

```diff
@@ -278,6 +278,34 @@ Foo.init({ /* attributes */ }, {
 });
 ```
 
+## Prevent creating a default PK attribute
+
+By default, Sequelize automatically adds the primary key attribute `id` to every model when there is no primary key has been defined manually. To prevent this you can use the option `noPrimaryKey` when creating the instance and set it to true.
```

Comment:
I left an extra `there is` during my previous review 😅 

Extra suggestion: reworded the second sentence because the example that follows doesn't show a creation of a Sequelize instance anymore but the creation of a model :)

```suggestion
By default, Sequelize automatically adds the primary key attribute `id` to every model when no primary key has been defined manually. To prevent this you can set the `noPrimaryKey` option to true when defining the model.
```